requirements.txt

```diff
@@ -1,11 +1,44 @@
+appdirs==1.4.3
+autopep8==1.5.4
 beautifulsoup4==4.9.3
 bs4==0.0.1
+CacheControl==0.12.6
+certifi==2019.11.28
+chardet==3.0.4
+click==7.1.2
+colorama==0.4.3
```

Comment:
colorama is already there at the bottom with version >0.4.4
might cause issue with this